README.md

```diff
@@ -59,11 +76,13 @@ evervault = Evervault::Client.new(api_key: <YOUR-API-KEY>)
 result = evervault.encrypt_and_run(<CAGE-NAME>, { hello: 'World!' })
 ```
 
-## API Reference
+## Reference
+
+The Evervault Ruby SDK exposes seven methods.
```

Comment:
I think what @jonnyom means is that, saying the actual _number_ of methods is not needed, and we have to always be keep an eye on it with every update, which can be error-prone. So we might as well change to:

```suggestion
These are the methods exposed by our Ruby SDK.
```

Or something like it?

Reply:
Something like it, we had a chat in engineering that it was mainly the phrasing rather than the number that was the core issue. Words like `at present` didn't sit correctly. Keeping on top of the number is an additional pain, but knowing how many methods there are can be useful too, I guess